### PR TITLE
CLOUDP-262239: Added a note on image SHAs for certified images

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

Added a note on image SHAs for certified images

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes-template.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: docs/dev/release.md [hunk 1]
```diff
@@ -142,12 +142,18 @@ The direct link, at the time of writing is https://connect.redhat.com/component/
 Copy the **certified** image SHAs of the **amd64** and the **arm64** image:
 
 ![img.png](certified-image-sha.png)
-
+> **NOTE**
+> 
+> In case you can't see the images tags in the https://connect.redhat.com go to the https://quay.io/repository/mongodb/mongodb-atlas-kubernetes-operator?tab=tags
+and select the latest released tag that ends with `-certified` e.g. `2.4.0-certified`. Then click on `manifest` link for the selected image.
+You should see two available manifests for `amd64` and `arm64` that you can use. Here is the example: https://quay.io/repository/mongodb/mongodb-atlas-kubernetes-operator/manifest/sha256:20740f06db3b6e768b40562ee50c729aba9870a9051045b4fc49d0c404cc18f6
+![img.png](img.png)
```
Comment (s-urbaniak) [inline docs/dev/release.md:150]: img.png does not seem to be part of this PR

Comment (josvazg) [reply docs/dev/release.md:150]: +1

**Review by josvazg — COMMENTED** [review]
Wording LGTM, but we seem to be missing an image file.

File: docs/dev/release.md (hunk 1 above)
Comment (igor-karpukhin) [reply docs/dev/release.md:150]: Omg, missed that. Fixed!

**Review by josvazg — APPROVED** [review]
